docs/maintainers/README.md

```diff
@@ -0,0 +1,70 @@
+
+## Concept Exercises
+
+The key things to understand about v3 is that we are making a strong distinction between Concept Exercises and Practice Exercises. All existing exercises will become Practice Exericses - exercises where students can practice the ideas they are learning, and discuss them with a mentor. The new Concept Exercises will be written from scratch, each with the aim of teaching one or more programming concepts that are necessary to become fluent in a given language.
+
+We want to crowd-source the creation of these Concept Exercises, providing clear guidelines and instructions for contributors. We are doing this by asking maintainers to create a GitHub Issue for each Concept Exercise.
+
+In order to do this maintainers need to understand what they think their tracks are going to look like. We have determined that the best way to do this is to start to list out all of the Concepts that someone needs to cover to learn a given language. Once this is done, you can start to turn each Concept into an issue that someone in the community can pick up and run with.
+
+The final step to preparing your language to accept contributions is to provide a sample Concept Exercise that you have written. Not only does this act as an example for other contributors, it is also crucial for you understanding the process and getting your teeth into the detail.
+
+We recommend choosing a Concept that involves working with basic types, like strings and numbers. Having chosen a Concept, look for the Concept's description in this reference part of this repository. This file should have references to all tracks that have implemented the exercise. You can use these existing implementations as a starting point for your first concept exercise, which you can then tailor to your specific language.
+
+### Read more
+
+- [More about Concepts and Concept Exercises](../concept-exercises.md)
+- [How to determine Concepts for your track](./determining-concepts.md)
+- [How to write Concept Exercise GitHub issues](./writing-a-concept-exercise-github-issue.md)
+- [How the repository is structured](./repository-structure.md)
+
+## Migrating your config.json files
+
+For v3, the existing `config.json` files will need to be updated. You can use the [C# config.json](../../languages/csharp/config.json) for reference.
+
+1. Add a version property:
+
+```json
+"version": 3
+```
+
+2. Add online editor settings:
+
+TODO: Write up something about https://github.com/exercism/ruby/blob/master/config.json#L7
+
+3. Convert the `"exercises"` array to an object with two properties:
+
+```json
+"exercises": {
+  "concept": [],
+  "practice": []
+}
+```
+
+4. Added concepts for all Practice Exercises
+
+The existing exercises are temporarily removed from the `config.json` file. They will return as practice exercises once the concept exercises have been added.
+
+TODO: Discuss more about what needs to be done
```

Comment:
@iHiD Do we need to write this bit already? I know what I want the new format to look like, so I could do that.